src/Subscriptions/Contracts/SubscriptionIterator.php

```diff
@@ -8,9 +8,18 @@
 interface SubscriptionIterator
 {
     /**
-     * Process collection of items.
+     * Process subscribers.
      *
+     * @param  \Illuminate\Support\Collection<\Nuwave\Lighthouse\Subscriptions\Subscriber>  $subscribers
+     * The subscribers that receive the current subscription.
+     *
+     * @param  \Closure  $handleSubscriber
+     * Receives each subscriber in the passed in collection.
+     * function(\Nuwave\Lighthouse\Subscriptions\Subscriber $subscriber): void
+     *
+     * @param  \Closure|null  $onError
+     * Is called when $handleSubscriber throws.
      * @return void
      */
-    public function process(Collection $items, Closure $cb, Closure $error = null);
+    public function process(Collection $subscribers, Closure $handleSubscriber, Closure $onError = null);
```

Comment:
Feel free to improve the naming and description of this interface and the parameters.